chore(ci): attempt to fix rare flake

ludamad opened the pull request:
I'm assuming this is a very rare time-of-check-vs-time-of-use race condition

```
stat: cannot statx '/tmp/b14e7e9201f291bc': No such file or directory
/home/aztec-dev/aztec-packages/ci3/run_test_cmd: line 108: 1756344688 -  : syntax error: operand expected (error token is "-  ")
``` 
in 
http://ci.aztec-labs.com/8927398b96150dd1

spalladino reviewed (approved):
I cannot parse that, but I'll trust you

Commit:
chore(ci): attempt to fix rare flake

I'm assuming this is a very rare time-of-check-vs-time-of-use race condition

```
stat: cannot statx '/tmp/b14e7e9201f291bc': No such file or directory
/home/aztec-dev/aztec-packages/ci3/run_test_cmd: line 108: 1756344688 -  : syntax error: operand expected (error token is "-  ")
```
in
http://ci.aztec-labs.com/8927398b96150dd1

Co-authored-by: ludamad <adam.domurad@gmail.com>

ludamad commented:
> I cannot parse that, but I'll trust you

and yet every time you say this, there was an embarrassing typo :)
